catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/DynamicSchemaResolver.java

```diff
@@ -437,7 +438,7 @@ private String createCenterPoint(List<Serializable> values) {
     return centerPoint.getY() + "," + centerPoint.getX();
   }
 
-  private byte[] createTinyBinary(String xml)
+  protected byte[] createTinyBinary(String xml)
```

Comment:
❓ Should we inject a `Function<byte[], String>` in the constructor and call that to create the tiny binary instead of making the method `protected` and have to override in the sub-class? If we don't do that, should we lower the visibility to package-private instead of `protected` since we only expect our tests to override?

Reply:
Which one is preferred? I forgot about package-private, that would be my inclination for simplicity's sake.

Reply:
Changing the visibility modifier feels much less invasive and equally less obscuring to the readability of the code as well.

Reply:
I agree. I would make it package private and annotate the method with `@VisibleForTesting`.